packages/modal/README.md

```diff
@@ -66,9 +66,9 @@ This code will render:
 
   > Whether `onRequestClose` should be called or not when the ESC key is pressed.
 
-- **showCloseIcon** (_Boolean_) = `true`
+- **showCloseButton** (_Boolean_) = `true`
 
-  > Whether the close icon should be visible or not.
+  > Whether the close button should be visible or not.
```

Comment:
```suggestion
  > Whether the close button should be visible or not. When visible, it will be rendered inside the `header` section whether there's a header or not.
```